app/routes/models/search/resultsModel.ts

```diff
@@ -162,30 +177,32 @@ const getDownloadResultsLink = (searchString: string, filterBy: any, sortBy: str
 
 const performSearch = async (searchString: string, requestedPage: number, filterBy: any, sortBy: string) => {
   const offset = (requestedPage - 1) * config.search.limit
-  const { results, total } = await getPaymentData(searchString, offset, filterBy, sortBy)
-
-  const matches = results.map((x: any) => ({...x, amount: getReadableAmount(parseFloat(x.total_amount))}))
+  const searchResults = await getPaymentData(searchString, offset, filterBy, sortBy)
   return {
-    matches,
-    total: total
+    ...searchResults,
+    results: searchResults.results?.map((x: any) => ({...x, amount: getReadableAmount(parseFloat(x.total_amount))}))
```

Comment:
Can this mapping be done outside and before this , so the readability improves?

Reply:
Sure, no worries